Intersect.Client/Core/Graphics.cs

```diff
@@ -768,6 +827,60 @@ public static void DrawFullScreenTextureFitMaximum(GameTexture tex)
             }
         }
 
+        public static void DrawFullScreenTextureAnimated(
+            string animFrameName,
+            long animFrameInterval,
+            Enum displayMode
+        )
+        {
+            sAnimFrameCount = Globals.ContentManager.GetTextureNames(TextureType.Animation)
+                .Count(s => s.Contains(animFrameName));
+
+            if (sAnimTextureTime < Timing.Global.Milliseconds)
+            {
+                sAnimTextureFrame += 1;
+                sAnimTextureTime = Timing.Global.Milliseconds + animFrameInterval;
+            }
+
+            if (sAnimTextureFrame >= sAnimFrameCount - 1)
+            {
+                sAnimTextureFrame = 0;
+            }
+
+            var animTex = Globals.ContentManager.GetTexture(
+                TextureType.Animation, animFrameName + "_" + sAnimTextureFrame + ".png"
+            );
+
+            if (animTex == null)
+            {
+                return;
+            }
+
+            switch (displayMode)
+            {
+                case DisplayModes.Default:
+                    DrawFullScreenTexture(animTex);
```

Comment:
Is it just me or does this method (`DrawFullScreenTexture`, it's already part of the code base) look like it's doing the exact same thing as "stretch" does, but... in extra steps?

@Arufonsu @Cheshire92 @jcsnider

Reply:
somehow it does and it doesn't?
i realized that if the texture itself is bigger than the game screen size, the 'default' DrawFullScreenTexture doesn't stretches down the texture, but it **does** stretches them up when the screen is bigger, so yeah they are similar but not exactly the same?